src/constants.ts

```diff
@@ -38,6 +38,8 @@ export interface ActionInterface {
   folderPath?: string
   /** Whether to force-push or attempt to merge existing changes. */
   force?: boolean
+  /** How many times to attempt to merge existing changes into the remote HEAD. */
+  attemptLimit?: number | null
```

Comment:
Is `| null` needed here if it's marked as optional?

```suggestion
  attemptLimit?: number;
```